src/pages/event/[id].js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Event from '../../components/Event';
+import getEvent from '../../db/getEvent';
+
```

Comment:
@jnbooth we're getting a build error from this file:
```
Error: getStaticPaths is required for dynamic SSG pages and is missing for '/event/[id]'.
```

Reply:
Oh yeah. Whoops. Fixed now. This is an optimization thing that Next.js can do, and I forgot we need to specifically opt out of it instead of just not using it.